UPGRADING.rst

```diff
@@ -10,6 +10,27 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#3497 Fix: API Gateway log is missing important fields
+======================================================
+
+To upgrade API Gateway logs format, Terraform needs to run the
+CloudWatch log group provisioner script. Run::
+
+    (cd terraform && terraform taint aws_cloudwatch_log_group.indexer && \
+    terraform taint aws_cloudwatch_log_group.service) && make deploy
+
+To troubleshoot errors regarding API Gateway log groups after performing the
+previous command run the following and perform steps three and four in quick
+succession.
```

Comment:
This sounds alarming but hand-wavy since the reader won't know why and what you define "quick succession" as.